deploy.ps1

```diff
@@ -180,21 +180,34 @@ $k8sReqs = @(
             kubectl apply -f https://github.com/weaveworks/kured/releases/download/1.2.0/kured-1.2.0-dockerhub.yaml
         }
     },
-    @{
-        Name     = "Harden Cluster"
-        Describe = "Apply security policy"
-        Test     = { kubectl get psp } # Improve tests
-        Set      = {
-            # Install the aks-preview extension
-            az extension add --name aks-preview
+    # @{
```

Comment:
Is this supposed to be commented out?

Reply:
Ya i haven't figured out how to merge it into the deploy script